src/ComputeManagement/Microsoft.WindowsAzure.Management.Compute.nuget.proj

```diff
@@ -5,7 +5,7 @@
     Microsoft.WindowsAzure.Management.Compute
     -->
     <SdkNuGetPackage Include="Microsoft.WindowsAzure.Management.Compute">
-      <PackageVersion>11.2.1</PackageVersion>
+      <PackageVersion>11.2.2</PackageVersion>
```

Comment:
If it's not a breaking change, and you are adding a new API, it shall be ```11.3.0```.

@markcowl Could you please help check?